comms/discovery/rpcz/websocket.go

```diff
@@ -96,3 +99,34 @@ func websocketPush(userId int32, payload []byte) {
 	}
 
 }
+
+func websocketPushAll(rpcJson json.RawMessage, timestamp time.Time) {
+	mu.Lock()
```

Comment:
for my own understanding - why is a lock needed here?

Reply:
nvm i see `websocketPush` does the same thing, I assume it's because websockets aren't thread safe?

Reply:
@stereosteve ? frankly i just copied code